src/components/faceVerification/screens/IntroScreen.jsx

```diff
@@ -97,7 +145,11 @@ const IntroScreen = ({ styles, screenProps, navigation }) => {
   const { fullName } = useProfile()
   const { showDialog } = useDialog()
 
-  const { isDelta, firstName, isFVFlow, isFVFlowReady } = useContext(FVFlowContext)
+  const { account: externalAccount, isDelta, firstName, isFVFlow, isFVFlowReady } = useContext(FVFlowContext)
+  const goodWallet = useWallet()
+  const { account } = goodWallet ?? {}
+  const [expiryDate, , state] = useIdentityExpiryDate(externalAccount || account)
```

Comment:
i couldnt find the code for this hook.
not sure what "state" is supposed to be

Reply:
https://github.com/GoodDollar/GoodWeb3-Mono/commit/1985097e96fe0d42788c2eaf81d3a2925c586650